Describe policy about support for OS versions used. (#21697)
mistercrunch/Airflow: README.md

* Previous versions [require](https://github.com/apache/airflow/issues/8162) at least Python 3.5.3
  when using Python 3.

## Base OS support for reference Airflow images

> The Airflow Community provides conveniently packaged container image that is published whenever
> we publish Apache Airflow release. Those images contain:

Reviewer: ```suggestion
The Airflow Community provides conveniently packaged container images that are published whenever
we publish an Apache Airflow release. Those images contain:
```
